vedro_gitlab_reporter/_gitlab_reporter.py

```diff
@@ -276,5 +284,5 @@ class GitlabReporter(PluginConfig):
     # Max stack trace entries to show (min=4)
     tb_max_frames: int = 8
 
-    # Show the relative path of each passed scenario
-    show_paths: bool = False
+    # Show the relative path of scenario in status
+    show_paths: list = [status.value for status in [ScenarioStatus.FAILED, ScenarioStatus.PASSED]]
```

Comment:
The proposed change from `show_paths: bool` to `show_paths: list` introduces a break in backward compatibility. To address this, we need to ensure the `show_paths=False` case is adequately handled.